README.md

```diff
@@ -1,16 +1,38 @@
-# lk2nd
-lk2nd is a bootloader for Qualcomm MSM devices (at the moment only MSM8916 and MSM8939),
-based on the [CodeAurora Little Kernel fork](https://source.codeaurora.org/quic/la/kernel/lk/).
-It provides an Android Fastboot interface on devices where the stock bootloader
-does not provide Fastboot (e.g. Samsung).
-
-On MSM8916 it is also used for some quirks for mainline devices, e.g. to set
-a proper WiFi/BT MAC address in the device tree.
-
-lk2nd does not replace the stock bootloader. It is packaged into an Android
-boot image and then loaded by the stock bootloader as a "secondary" bootloader.
-The real Android boot image is placed into the boot partition with 512 KiB offset,
-and then loaded by lk2nd.
+# lk2nd (and lk1st)
```

Comment:
Maybe call the project lk2nd and refer to lk1st as it's configuration variant? Thus the first paragraph will refer to lk2nd as a bigger entity that includes lk1st and whatever else?

They are *the same* bootloader, just configured to boot differently.